DateRangePicker/DateRangePickerView.swift

```diff
@@ -166,11 +166,16 @@ open class DateRangePickerView: NSControl, ExpandedDateRangePickerControllerDele
 	}
 	
 	// MARK: - Methods
+    open func makePopover() -> NSPopover {
```

Comment:
This is still indented with spaces instead of tabs. (Have a look at GitHub's review view if you don't believe me.)